lapack/testlapack/dgeev.go

```diff
@@ -658,80 +643,79 @@ func testDgeev(t *testing.T, impl Dgeever, tc string, test dgeevTest, jobvl lapa
 		return
 	}
 
-	// Check that the columns of VL and VR are eigenvectors that correspond
-	// to the computed eigenvalues.
-	for k := 0; k < n; {
-		if wi[k] == 0 {
-			if jobvl == lapack.LeftEVCompute {
-				ev := columnOf(vl, k)
-				if !isLeftEigenvectorOf(test.a, ev, nil, complex(wr[k], 0), vecTol) {
-					t.Errorf("%v: VL[:,%v] is not left real eigenvector",
-						prefix, k)
-				}
+	// Check that the columns of VL and VR are eigenvectors that:
+	//  - correspond to the computed eigenvalues
+	//  - have Euclidean norm equal to 1
+	//  - have the largest component real
+	bi := blas64.Implementation()
+	if jobvr == lapack.RightEVCompute {
+		resid := residualRightEV(test.a, vr, wr, wi)
+		if resid > vecTol {
+			t.Errorf("%v: unexpected right eigenvectors; residual=%v, want<=%v", prefix, resid, vecTol)
```

Comment:
Currently prefix uses the `%v` verb for the `jobvr` and `jobvl` values. This should probably be `%c` for ease of reading, but that would be another PR since there are a bunch of them and they should all change together.

Reply:
Ok, I'll put it on my todo list.